src/__tests__/store/feed/reducer.test.ts

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { feedReducer } from '../../../store/feed/reducer'
+import { IFeedState, FeedActionTypes } from "../../../store/feed/types";
+import { any } from 'prop-types';
+import { IFeedItem } from "../../../api/models/feed.model";
+import { IPluginItem } from "../../../api/models/pluginInstance.model";
+
+const TestIFeedItem: IFeedItem = {
```

Comment:
TestFeedItem is better here. I is for interface.

Reply:
Sure